currency_rate_update_nbu/README.rst

```diff
@@ -0,0 +1,108 @@
+========================
```

Comment:
readme is to be added to a separate folder and split into parts, this README.rst will be regenerated by the bot after the merge 
example 
https://github.com/OCA/l10n-ukraine/tree/12.0/account_bank_statement_import_online_ua_pb_interpay/readme

Reply:
Thanks! Implemented.